### ISSUE TITLE
Import failure

### ISSUE THREAD

**Issue by delaneyj** [gorgonia/tensor#29]
`go get -u "github.com/gorgonia/tensor"`

return an error of 
```
package github.com/gorgonia/tensor: code in directory C:\go\src\github.com\gorgonia\tensor expects import "gorgonia.org/tensor"
```

**Comment by chewxy** [comment]
You need to `go get -u gorgonia.org/tensor`

**Comment by delaneyj** [comment]
Thanks, just getting started and [the README](https://github.com/gorgonia/tensor#usage) says otherwise.

**Comment by chewxy** [comment]
Ah yes. It should be fixed. Feel free to send a PR. Otherwise it'll get rolled in to the v0.9.0 fixes

**Comment by chewxy** [comment]
Fixed in v0.9.0 branch
